doc/examples/segmentation/plot_regionprops.py

```diff
@@ -97,32 +97,44 @@
 mask = morphology.remove_small_holes(mask, 50)
 labels = measure.label(mask)
 
+property_names = ["label", "area", "eccentricity", "perimeter", "intensity_mean"]
+
+props = pd.DataFrame(measure.regionprops_table(labels, img, properties=property_names))
```

Comment:
We already have another [example, `plot_regionprops_table.py`](https://github.com/scikit-image/scikit-image/blob/107c1a9a2c49f86668c41e8e5a1c101d71ff122f/doc/examples/segmentation/plot_regionprops_table.py#L4), that showcases `regionprops_table`. This example should probably keep using `regionprops` directly.

Do you want to try a solution without using `regionprops_table` and `pandas`?

Reply:
done!